MainApp/templates/main_page.html

```diff
@@ -0,0 +1,41 @@
+{% extends 'base.html' %}
+
+{% block title %}
+    Main page
+{% endblock %}
+
+{% block head %}
+    {% load static %}
```

Comment:
template tags should be on the first line of the file. Move {% load static %} to the top please